doc/set_fees_process.md

```diff
@@ -0,0 +1,40 @@
+# Setting fees process
+
+It will be explained on an example of Indy CLI.
+
+### Prerequisites
+
+* libindy
+* Indy CLI
+* libsovtoken
+* Wallet with Trustee DID
+* Created pool in Indy CLI
+
+[Small guide](https://github.com/hyperledger/indy-sdk/tree/master/doc/design/001-cli#commands) to Indy CLI commands: 
+
+### Creating a SET_FEES transaction.
+
+* Open `indy-cli`
+* Open wallet: `wallet open <wallet_name> key=<wallet_encryption_key>`
+* Load libsovtoken: `load-plugin library=libsovtoken.[so|dll] initializer=sovtoken_init`
+* Make a transaction with a command `ledger set-fees-prepare payment_method=sov fees=NYM:1` to set fees to 1 sovatom for NYM
+
```

Comment:
I think it might be good to have an example with fees set for multiple transactions

Reply:
I think I will follow Darien's advice and add examples for each step